src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PerformTouchAction.java

```diff
@@ -0,0 +1,360 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.mercury;
+
+import java.util.List;
+
+import io.appium.java_client.MultiTouchAction;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.android.AndroidDriver;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import com.wikia.webdriver.common.driverprovider.NewDriverProvider;
+
+/**
+ * @authors: Tomasz Napieralski
+ * @created: 9 Jan 2015
+ * @updated: 20 Jan 2015
+ */
+
+public class PerformTouchAction {
+	
+	private static WebDriver driver;
+	private static AndroidDriver mobileDriver;
+	
+	private static int nativeHeight = 0;
+	private static int nativeWidth = 0;
+	private static int webviewHeight = 0;
+	private static int webviewWidth = 0;
+	private static int ratio = 0;
+	
+	private static int taskbarNativeHeight = 0;
+	private static int taskbarNativeWidth = 0;
+	private static int taskbarWebviewHeight = 0;
+	private static int taskbarWebviewWidth = 0;
+	
+	private static int appNativeHeight = 0;
+	private static int appNativeWidth = 0;
+	private static int appWebviewHeight = 0;
+	private static int appWebviewWidth = 0;
+	
+	private static int loadedPageHeight = 0;
+	private static int loadedPageWidth = 0;
```

Comment:
What is the reason that this variable needs to be static?
Normally, we want to definite it with 'final' to be 'static final'.